src/mgos_imu_icm20948.c

```diff
@@ -0,0 +1,482 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mgos.h"
+#include "mgos_i2c.h"
+#include "mgos_imu_icm20948.h"
+
+#define ACC_BASE_ODR  1125.f
```

Comment:
These two defines should be scoped as MGOS_ICM20948_* to avoid defines clashing in other codebases. Please rename.

Reply:
In my understanding, defines placed in source file are not visible anywhere else, that's why I used short names, but I could be wrong.

Reply:
You're right.